angular-universal-ssr/host-app/server.js

```diff
@@ -3,5 +3,5 @@
 const mainModule = __non_webpack_require__.main;
 const moduleFilename = (mainModule && mainModule.filename) || '';
 if (moduleFilename === __filename || moduleFilename.includes('iisnode')) {
-  import('./server-bootstrap').then(x => x.run());
+  import('./server-bootstrap.ts').then(x => x.run());
```

Comment:
host-app on localhost:64281 has errors 
<img width="1661" alt="Screenshot 2023-01-12 at 19 31 34" src="https://user-images.githubusercontent.com/55971575/212138381-ff746dd5-fc14-42e3-b00e-a94d33f5d1b5.png">

Please take a look)

Reply:
Where did you get this port from?